knotx-core/src/main/java/io/knotx/fragments/FragmentContentExtractor.java

```diff
@@ -27,22 +27,41 @@ private FragmentContentExtractor() {
     //util class
   }
 
+  /**
+   * Extracts pure markup from the given {@link Fragment}'s content removing Knotx data.
```

Comment:
Ok, changed. I didn't want to use `script` word originally because we may want to change `<script>` tag to something different.